aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -99,6 +99,14 @@ Tensor expand_batching_rule(const Tensor& self, IntArrayRef size, bool implicit)
   return self_physical.newLogicalFromPhysical(result);
 }
 
+std::vector<Tensor> chunk_batching_rule(const Tensor& self, int64_t chunks, int64_t dim) {
+  auto self_physical = MultiBatchVmapTransform::logicalToPhysical(self);
+  auto dim_physical = self_physical.getPhysicalDim(dim);
+  auto result = at::chunk(self_physical.tensor(), chunks, dim_physical);
+  self_physical.makeLogicalFromPhysicalListInplace(result);
+  return result;
```

Comment:
I wonder if we should try to golf this some more LOL. A lot of operators like this, where they only need a dim translation.

Reply:
Yeah, we could have codegen or some metaprogramming do that. Will add it to a to-investigate list. I've thought about having generating batching rules from something like a classification of operators so such an approach could fit in there.